docker/Dockerfile.cpu

```diff
@@ -0,0 +1,56 @@
+##############################################################
+# This Dockerfile contains the Devito codes and can be buld using different base images.
+##############################################################
+
+ARG arch=gcc
+
+# Base image 
+FROM python:3.7 as base
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# Install for basic base not containing it
+RUN apt-get update && apt-get install -y mpich libmpich-dev libgl1-mesa-glx
+
+##############################################################
+# GCC standard image
+##############################################################
+FROM base as gcc
+# Env vars defaults
+ENV DEVITO_ARCH="gcc"
+ENV DEVITO_LANGUAGE="openmp"
+
+##############################################################
+# ICC image
+##############################################################
+FROM base as icc
+
+ # download the key to system keyring
```

Comment:
Download

Reply:
Can you add a comment with a link pointing to where these instructions have been lifted from?

Reply:
empty space at the start of line 28?